packages/pwa/app/commerce-api/auth.js

```diff
@@ -79,23 +78,54 @@ class Auth {
     }
 
     get authToken() {
-        return this._authToken
+        return this._storage.get(tokenStorageKey)
+    }
+
+    set authToken(token) {
+        this._storage.set(tokenStorageKey, token)
     }
 
     get refreshToken() {
-        return this._refreshToken
+        return (
+            this._storage.get(refreshTokenStorageKey) ||
+            this._storage.get(refreshTokenGuestStorageKey)
+        )
+    }
+
+    set refreshToken(obj = {}) {
```

Comment:
To keep in consistent with the SLAS cartridge, we need to store guest/registered refresh token separately.

Reply:
Why is being consistent with the SLAS cartridge a goal?

Reply:
It'll make phased rollout easier. Both SLAS cartridge and PWA read/write to the same cookies.